core/modules/ccontrol/UserQuerySelect.cc

```diff
@@ -162,7 +162,7 @@ std::string UserQuerySelect::getError() const {
 
 /// Attempt to kill in progress.
 void UserQuerySelect::kill() {
-    LOGS(_log, LOG_LVL_DEBUG, "UserQuerySelect kill");
+    LOGS(_log, LOG_LVL_DEBUG, _queryIdStr << " UserQuerySelect kill");
```

Comment:
I'd probably use `getQueryIdString()` instead of `_queryIdStr ` to log it. If someone subclasses `UserQuerySelect` and overrides `getQueryIdString()` you probably want log output be consistent here and in the clients of this class.